src/App/Components/Loader/Loader.js

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import { ClimbingBoxLoader } from 'react-spinners';
+
+
+const Loader = () => {
```

Comment:
Kindly switch to the loader that was agreed upon by the team? @MCFrank16

Reply:
I am still waiting for it to be merged and then use it, or if it is possible pass it on to me

Reply:
you can take from my PR to avoid future delays